### ISSUE TITLE
bump to actual version (6.09-ucs1 --> 7.09+)

### ISSUE THREAD

**Issue by afgym-baeu** [wekan/univention#22]
Latest version in UCS app center catalog is 6.09-ucs1. Seems it did not get an update for about 20 month.

Please bump to 7.09 or later.

Thanks a lot.

**Comment by xet7** [comment]
@afgym-baeu 

Yes, after I have fixed migrations so that they work properly, hopefully this year.

**Comment by fridok** [comment]
@xet7 Are there any news on that issue? Latest Wekan version in UCS app center is still 6.09-ucs-1. I would love to use the new features.

**Comment by xet7** [comment]
@fridok 

Well, maybe this month Snap Stable will update. Maybe update for UCS at next month. Let's see how it goes.

**Comment by fridok** [comment]
@xet7 Any news on a new version für UCS? Still UCS is at 6.09-ucs1 while current version is at 7.45.

**Comment by xet7** [comment]
@fridok 

Still In Progress:
- Some settings need to be figured out and added, so that WeKan would work
- Needs additional code, so that migrating database would work

When I get it working, I try to release it.

Another way, that is also currently in testing, is to add newest WeKan with name WeKan 2, so that new users could start with WeKan 2. And later provide some way to migrate from old version.

**Comment by afgym-baeu** [comment]
@xet7 

Any progress the one or the other way?

Thanks for your effort!

**Comment by afgym-baeu** [comment]
It would just be great to get an upgrade path or a parallel "fresh" release without an upgrade path.

**Comment by xet7** [comment]
Hi,
creating WeKan update for UCS is in development,
but it may take some time before it is released,
because it requires adding support for
also old versions of MongoDB,
so there would not be required any time migrating
database, it would work with existing old database.
This is because migrating database to newer
in many cases can take too long - many hours,
and at some cases there is not enough disk space
to migrate to different version of database.
When upcoming version of WeKan is ready,
and works also with existing database,
then I will release new WeKan version for UCS
and other platforms.
I try to have updates available for all platforms
ASAP, but that probably is not today yet.

**Comment by fridok** [comment]
@xet7 Is there any news on an upcoming update to the now very old version of Wekan in UCS?

**Comment by xet7** [comment]
@fridok 

Yes. There will be upgrades for all WeKan platforms. Developing those upgrades is In Progress. More details here:

https://wekan.fi/upgrade/
